src/worker.js

```diff
@@ -8,11 +8,16 @@ import BatchManager from './utils/BatchManager';
 
 let closing = false;
 
+const healthCheckEndpoint = '/';
+
 const attachMiddleware = (app, config) => {
   app.use(bodyParser.json(config.bodyParser));
 };
 
 const attachEndpoint = (app, config, callback) => {
+  app.get(healthCheckEndpoint, function (req, res) {
+    res.send('It works!\n');
```

Comment:
Different folks will have different requirements for how the endpoint should behave which is why I think we should keep this in userland.